manifests/init.pp

```diff
@@ -17,7 +17,11 @@
 class cobald(
   String                     $cobald_version             = undef,                        # cobald version to be used (undef = latest PyPI release, 'master' = Github master branch or PyPI release number)
   String                     $tardis_version             = undef,                        # tardis version to be used (undef = latest PyPI release, 'master' = Github master branch or PyPI release number)
+  Enum['PyPI', 'git']        $cobald_provider            = 'PyPI',                     # Cobald source provider (PyPI or Git)
+  Enum['PyPI', 'git']        $tardis_provider            = 'PyPI',                     # TARDIS source provider (PyPI or Git)
```

Comment:
```suggestion
  Enum['pypi', 'git']        $cobald_repo_type           = 'pypi',                     # cobald repository type (PyPI or git)
  Enum['pypi', 'git']        $tardis_repo_type           = 'pypi',                     # tardis repository type (PyPI or git)
```
I think the word "provider" is misleading here. I associate the entity from which one fetches the code with "provider". It also has a special meaning in the Puppet context.